.evergreen/.evg.yml

```diff
@@ -1916,15 +1916,15 @@ axes:
       - id: "2.11"
         display_name: "Scala 2.11"
         variables:
-          SCALA: "2.11.12"
```

Comment:
`libs.versions.toml` now handles the scala library versioning. So this is now just for which scala release to test against.